### PR TITLE
PasswordGenerator.py

### PR DESCRIPTION
**Pull request by 6Infected9**
Creates a Strong Password for fighting against bruteforce attack.

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: PasswordGenerator.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# The main purpose of this is to get secure password to defend against Bruteforce attacks.
+
+
+import string
+import random
+import re
+
+if __name__ == '__main__':
```
Comment (L3str4nge) [inline PasswordGenerator.py:8]: What kind of algorithm is that?

Comment (cclauss) [reply PasswordGenerator.py:8]: The algorithm should be a function that returns a password (str) and does not print().  See CONTRIBUTING.md

File: PasswordGenerator.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# The main purpose of this is to get secure password to defend against Bruteforce attacks.
```
Comment (cclauss) [inline PasswordGenerator.py:1]: This repo requires that files are placed in an existing directory.  See CONTRIBUTING.md.

**Comment by cclauss** [comment]
Also, PasswordGenerator.py --> password_generator.py

**Comment by poyea** [comment]
Related PR #4950

**Pull request closed without merging**